maintainers/maintainer-list.nix

```diff
@@ -9031,4 +9031,10 @@
     github = "saulecabrera";
     githubId = 1423601;
   };
+  tfmoraes = {
+    name = "Thiago Franco de Moraes";
+    email = "351108+tfmoraes@users.noreply.github.com";
+    github = "tfmoraes";
+    githubId = 351108;
+  };
```

Comment:
Please make this into a separate commit:

```
maintainers: add <name>
```

Reply:
Done!